tests/helpers.bash

```diff
@@ -296,6 +296,16 @@ function rm() {
     run_unshared rm "$@"
 }
 
+#################
+#  run_with_log  #  Logs command before running it
+#################
+#
+function run_with_log() {
+    local cmd="$*"
+    echo "$_LOG_PROMPT $cmd"
+    run -0 $cmd
```

Comment:
This breaks quoting of arguments, you need to use `run -0 "$@"` here to ensure the args always work correctly

Likely doesn't matter for the cases where you added it but if we add as general helper we should fix it.